app/common/data/interfaces/temporary.py

```diff
@@ -51,19 +52,39 @@ def delete_collection(collection_id: UUID) -> None:
     db.session.flush()
 
 
-def delete_section(section_id: UUID) -> None:
-    section = db.session.query(Section).where(Section.id == section_id).one()
+def delete_section(section_id: UUID, collection_id: UUID) -> None:
```

Comment:
These should all just take the DB object `section` itself, that feels like maybe a nicer interface than passing a raw ID? I think we're likely to always have that object loaded already? Then we don't need to do a query to fetch the collection, we can just do `section.collection`

Reply:
(but they're just temp/dev interfaces you're right)